GageStatsServices/appsettings.json

```diff
@@ -13,7 +13,7 @@
     "pathDirectory": "Directory path to linked descriptions",
     "parameters": {
       "id": {
```

Comment:
might want to consider using a different key than 'id' to reflect your changed variable for station. The id parameter key description  in appsettings will be used for every endpoint that uses the 'id' variable. For example [agency ](https://test.streamstats.usgs.gov/docs/gagestatsservices/#/Agencies/GET/Agency)and [edit agency](https://test.streamstats.usgs.gov/docs/gagestatsservices/#/Agencies/PUT/EditAgency) 'id' description will also change

Reply:
Oh, I didn't think about that. Thanks, I was just replicating the regions endpoint from NSSServices.